stdlib/string.pyi

```diff
@@ -1,4 +1,5 @@
-from typing import Any, Iterable, Mapping, Sequence, Tuple
+# from re import RegexFlag  # Importing from regex breaks the Markdown stubtest
+from typing import Any, Iterable, Mapping, Pattern, Sequence, Tuple
```

Comment:
That's hopefully fixed now if you merge in master. Also, `Pattern` should be imported from `re`, not `typing`.